tests/unit/genomics/test_runbigscape.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from nplinker.genomics.bigscape.runbigscape import run_bigscape
+from .. import DATA_DIR
+
+
+class TestRunBigscape:
```

Comment:
Using test class `class TestRunBigscape` is not necessary here. The code would be simpler if using test functions directly.